include/ck_tile/ops/epilogue/cshuffle_epilogue.hpp

```diff
@@ -65,8 +65,12 @@ struct CShuffleEpilogue
     static constexpr index_t kNPerXdl                      = Problem::kNPerXdl;
     static constexpr index_t kKPerXdl                      = Problem::kKPerXdl;
     static constexpr index_t isCTransposed                 = Problem::isCTransposed;
-    static constexpr index_t kMPerIteration                = kMPerXdl * kMWave;
-    static constexpr index_t kNPerIteration                = kNPerXdl * kNWave;
+
+    static constexpr index_t CShuffleMXdlPerWavePerShuffle = 1;
+    static constexpr index_t CShuffleNXdlPerWavePerShuffle = 2;
```

Comment:
What about such renaming? 
```suggestion
    static constexpr index_t NumMXdlPerWavePerShuffle = 1;
    static constexpr index_t NumNXdlPerWavePerShuffle = 2;
```

Reply:
Done.